### PR TITLE
ActivityFeedServletTest addition

### PR DESCRIPTION
**Pull request by abeltran1804**
Added a simple test case for the ActivityFeedServlet.java code. It simple calls the doGet() method to make sure it receives both convos and users. Also converted the time for nicer display.

### CODE REVIEW

File: src/test/java/codeu/controller/ActivityFeedServletTest.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        
+        
+        mockUserStore = Mockito.mock(UserStore.class);
+        activityFeedServlet.setUserStore(mockUserStore);
+    }
+    
+    @Test
+    public void testDoGet() throws IOException, ServletException {
+        List<Conversation> fakeConversationList = new ArrayList<>();
+        fakeConversationList.add( new Conversation(UUID.randomUUID(), UUID.randomUUID(), "test_conversation", Instant.now()));
```
Comment (acolwell) [inline src/test/java/codeu/controller/ActivityFeedServletTest.java:84]: Change second parameter to USER_ONE.getId() so it is owned by a user in the user list?

File: src/main/webapp/WEB-INF/view/activity.jsp [hunk 2]
```diff
@@ -49,9 +51,12 @@
...
         for(Conversation conversation : conversations){
       %>
-        <li> 
-         
-            <a> <%= conversation.getTitle() %> created at <%= conversation.getCreationTime()%> </a>
+         <li> 
+          <% DateTimeFormatter formatter = DateTimeFormatter.ofLocalizedDateTime( FormatStyle.SHORT ).withLocale( Locale.US ).withZone( ZoneId.systemDefault() );
+          Instant instant = conversation.getCreationTime();
+          String output = formatter.format( instant ); %>
+          <a> <%= conversation.getTitle() %> created at
```
Comment (acolwell) [inline src/main/webapp/WEB-INF/view/activity.jsp:58]: These links don't have hrefs. Are you planning on adding these in a followup change? If not, I'd say remove the <a> tags.

File: src/main/webapp/WEB-INF/view/activity.jsp [hunk 3]
```diff
@@ -27,7 +29,7 @@
       <a href="/login">Login</a>
     <% } %>
     <a href="/about.jsp">About</a>
-    <a href="/activity.jsp">Activity</a>
+    <a href="/activity">Activity</a>
```
Comment (acolwell) [inline src/main/webapp/WEB-INF/view/activity.jsp:32]: Good catch. I happened upon this very issue this morning and was going to IM you about it. :)

**Review by acolwell — APPROVED** [review]

File: src/main/webapp/WEB-INF/view/activity.jsp (hunk 2 above)
Comment (abeltran1804) [reply src/main/webapp/WEB-INF/view/activity.jsp:58]: Oh right, I was debating if they should link to anything or not. The chat events would link to chats and the users would link to the user's page. But another thing about it is I wasn't quite sure how to print variables. 
I tried used this:
https://stackoverflow.com/questions/23803440/how-to-display-a-variable-value-in-jsp-file
But ${variableName} didn't quite work nor did <% varibaleName; %> so I just left it as is. That being said, what' the official way of printing variables?

File: src/test/java/codeu/controller/ActivityFeedServletTest.java (hunk 1 above)
Comment (abeltran1804) [reply src/test/java/codeu/controller/ActivityFeedServletTest.java:84]: Fixed, thanks!

File: src/main/webapp/WEB-INF/view/activity.jsp (hunk 2 above)
Comment (acolwell) [reply src/main/webapp/WEB-INF/view/activity.jsp:58]: I think you might have missed a '='. I'd expect variable printing to look like <%= variableName %>. Notice the '=' after the '%'. I expect the link here to look like the links on the conversations page (https://github.com/aylinev2/CodeU-Summer-2018/blob/master/src/main/webapp/WEB-INF/view/conversations.jsp#L75)

**Pull request merged**